src/ORM/FieldType/DBVarchar.php

```diff
@@ -117,12 +123,14 @@ public function RTF()
 
     public function scaffoldFormField($title = null, $params = null)
     {
-        if (!$this->nullifyEmpty) {
-            // Allow the user to select if it's null instead of automatically assuming empty string is
-            return new NullableField(new TextField($this->name, $title));
-        } else {
-            // Automatically determine null (empty string)
-            return parent::scaffoldFormField($title);
+        // Set field with appropriate size
+        $field = TextField::create($this->name, $title);
```

Comment:
this is preferable to redefine the behaviour than to use `parent::scaffoldFormField($title, $params)`?

Reply:
I use this because it's a definite TextField, but scaffoldFormField only hints FormField.